Windows configuration path resolution bug

<!-- Here is for bug reports and feature requests ONLY! 

If you're looking for help, please check our Dingtalk group and the Wechat group.

Please try to use English to describe your issue, or at least provide a snippet of English translation.
-->

## Issue Description

Type: *bug report*

### Describe what happened
When using Jupiter, the following error occurs when Windows launches the specified configuration file:
panic: 
    msg: new engine
    errVerbose: runtime error: slice bounds out of range [:-1]
github.com/douyu/juno-agent/vendor/github.com/douyu/jupiter.(*Application).loadConfig-fm

### Describe what you expected to happen
Can support Windows system specified configuration file startup

### How to reproduce it (as minimally and precisely as possible)
1.  Execute the following command at the terminal: juno-agent --config=../../config/config.live.toml


### Tell us your environment
os: windows 8.1
go: 1.13.9

### Anything else we need to know?

--- a/pkg/util/xfile/file.go
+++ b/pkg/util/xfile/file.go
@@ -18,6 +18,7 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"runtime"
 	"strings"
 )
 
@@ -155,6 +156,9 @@ func substr(s string, pos, length int) string {
 }
 
 func getParentDirectory(dirctory string) string {
+	if runtime.GOOS == "windows" {
+		dirctory = strings.Replace(dirctory, "\\", "/", -1)
+	}
 	return substr(dirctory, 0, strings.LastIndex(dirctory, "/"))
 }
 

